client/src/pages/choosePlayers.tsx

```diff
@@ -76,57 +87,24 @@ export default function ChoosePlayers() {
     <div>
       <div>Choose Players</div>
       <h2>Names</h2>
-      {names
-        .filter((name) => !name.selected)
-        .map((name) => (
-          <Card
-            style={{ border: 'solid blue', margin: '.25em', width: '50em' }}
-            key={name.name}
-            onClick={() => handleClickName(name)}
-          >
-            <CardContent>
-              <Typography>{name.name}</Typography>
-            </CardContent>
-          </Card>
-        ))}
-      <div>
-        <p>Selected Names</p>
-        {names
-          .filter((name) => name.selected)
-          .map((name) => (
-            <Card
-              style={{ border: 'solid blue', margin: '.25em', width: '50em' }}
-              key={name.name}
-              onClick={() => handleClickName(name)}
-            >
-              <CardContent>
-                <Typography>{name.name}</Typography>
-              </CardContent>
-            </Card>
-          ))}
-      </div>
+      {musicians.map((musician: Name) => (
+        <Card
+          className={`${classes.card} ${musician.selected && classes.selected}`}
```

Comment:
do you think this would be a good case for the 'classNames' library? https://github.com/JedWatson/classnames